epsagon/trace_transports.py

```diff
@@ -23,7 +23,7 @@ class LogTransport(object):
 
     @classmethod
```

Comment:
don't make it a class method for no reason. It can be a static method if you don't use self but the owner of this method is the instance. (Also we should have a united interface - it can't be a classmethod here but a regular method in the HTTPTransport)

Reply:
also I would call it PrintTransport (since you use print and not logging) but doesn't matter